modules/api/scripts/instance_startup.sh

```diff
@@ -7,3 +7,10 @@ docker run -d \
   -e SLICK_CLICKHOUSE_URL='${SLICK_CLICKHOUSE_URL}' \
   -e ELASTICSEARCH_HOST='${ELASTICSEARCH_HOST}' \
   quay.io/opentargets/platform-api:${PLATFORM_API_VERSION}
+
```

Comment:
Are we not using the API metadata environment values?

Reply:
I might need your help with this one. For some reason when the docker container with the API starts it crashes when the API metadata variables are set. 

Based on the documentation I should be able to do something like this in the API configuration file:

```
ot.meta.dataVersion {
              year = 22
              year = ${?META_DATAVERSION_YEAR}
              month = 11
              month = ${?META_DATAVERSION_MONTH}
              iteration = 0
              iteration = ${?META_DATAVERSION_ITERATION}
            }
```
And those values should be picked up from the environment. 

I think the crash might be caused because of a datatype mismatch, but I haven't got it working yet.